assets/js/components/settings/settings-module.js

```diff
@@ -323,26 +322,16 @@ class SettingsModule extends Component {
 						>
 							<div className="mdc-layout-grid">
 								<div className="mdc-layout-grid__inner">
-									{ setupComplete && (
-										<Fragment>
-											<div className="
-												mdc-layout-grid__cell
-												mdc-layout-grid__cell--span-12
-											">
-												<SettingsRenderer
-													slug={ slug }
-													isEditing={ isEditing[ moduleKey ] }
-													isOpen={ isOpen }
-												/>
-											</div>
-										</Fragment>
-									) }
-									{
-										hasSettings && ! setupComplete &&
-											<ModuleSetupIncomplete
-												slug={ slug }
-											/>
-									}
+									<div className="
+										mdc-layout-grid__cell
+										mdc-layout-grid__cell--span-12
+									">
```

Comment:
Use `<Cell>` component instead. Also, please, update `<div className="mdc-layout-grid">` to be `<Grid>` and `<div className="mdc-layout-grid__inner">` to be `<Row>`